data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+				goto "that's awful"
+			label "skipped tours"
+			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall, "Yesterday a marauder terrorised the inhabitants of Allhome. They arrived by posing as a trader to investigate the rumours. Hai security failed to identify the danger and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
+				goto "that's awful"
+			label "that's awful"
+			`	At a table nearby you hear a Hai staff member exclaim, "Oh that's awful!"`
+			`	The rumours must be spreading further by the day you think, no doubt something will soon have to be done.`
+			`	This thought has no sooner crossed your mind when your solitude is interrupted by a small group of somewhat official-looking Hai.`
+			`	"Are you Captain <last>?" one of them asks you, "Don't worry, you're not in trouble or anything."`
+			`	You nod your affirmation.`
+			`	"Your reputation precedes you Captain. As a person of note at home who we hear has now been travelling around Hai space, you are presently being treated as an Envoy."`
+			`	For a second you wonder if it's a joke but they look as serious as Hai can look.`
+			choice
+				`	"Oh I really don't think that's necessary."`
+					goto nevertheless
+				`	"Oh, I'm honoured, thank you."`
+			`	"If you have any stake in the current information crisis please head directly to Hai-home. Someone would very much like to talk to you."`
+				goto main
+			label nevertheless
+			`	"You may not think it's necessary, but sometimes that is the price of fame."`
+			`	The officious-looking Hai shrugs, "In any event please head directly to Hai-home. Someone would very much like to talk to you."`
+			label main
+			`	They leave without waiting for your agreement, but then, you did come here to see what was going on.`
+				accept
+
+
+
+#All prior branches merge here.
+	
+mission "Hai Leaks Response 1"
+	landing
+	name "Hai message to <planet>"
+	description "Take the diplomatic ID from the box from Sayari to <destination>."
+	source "Hai-home"
+	destination "Earth"
+	to offer
+		or
+			has "Hai Tour [5] Tour Ends: done"
+			has "ZZ Event Hai Secret Leaks: RSHU: done"
+			has "ZZ Event Hai Secret Leaks: FWHU: done"
+	on offer
+		log "People" "Sayari" `Ambassador Sayari, a Hai, served as their envoy to the human government a century ago (although no public records of this exist in human space). Her experience has placed her in charge of coordinating the Hai response to the crisis of their continued secrecy.`
```

Comment:
This is the one spot as I recall that introduces a real quirk with existing saves. Sayari's log entries will be out of order and one of them won't strictly reflect what the current state is. However, for new saves they'll occur in order because the planet you land on to start wanderers won't let you land now until you complete an interaction in HR6.

Reply:
Interesting. I don't know if there's really a good resolution for this issue, but I imagine most of our core playerbase using the same file over multiple releases wouldn't mind _that_ much, especially since there's no simple fix.